tomviz/python/Recon_SIRT.py

```diff
@@ -92,19 +90,19 @@ def transform_scalars(self, dataset, Niter=10, stepSize=0.0001,
 
                 # Give 4 updates for first iteration.
                 if Nupdates != 0 and i == 0 and (s + 1) % (Nslice//4) == 0:
-                    utils.set_array(child, recon)
+                    child.active_scalars = recon
```

Comment:
This too, I think this is a big improvement, and I assume behind the scenes things are working in the same sense as they were before with updates/live updates.

Reply:
Yep! That is right. I made sure live updates are working, and the setting of the final recon dataset as well.